payload_designer/systems.py

```diff
@@ -113,9 +115,35 @@ def get_signal_to_noise(self, radiance: LUT, wavelength):
 
         return snr
 
+    def get_FOV_vector(self):
+        """Get the field of view vector. A vector that defines the angular
+        extent that can be imaged by the payload in the along-track and the
+        across-track directions.
+
+        Pre-condition:
+        self.foreoptic != None
+        self.slit != None
+
+        """
+        if self.slit is None or self.foreoptic is None:
+            return Exception("slit and foreoptic components cannot be NoneType.")
+
+        # first reshape the foreoptic vector so it is x by 1
+        x = self.foreoptic.focal_length.shape[0]
+        focal_lengths = (1/self.foreoptic.focal_length).reshape((x, 1))
+
+        # now reshape the slit vector so it is 2 x 1
+        slit_vector = np.array(self.slit.size)
+        # transpose it to a 1 x 2 vector
+        slit_vector = self.slit.size.reshape((2, 1)).T
+
+        # should return a vector that is x by 2
+        return np.degrees((2 * np.arctan(focal_lengths * slit_vector)).reshape((x, 2)))
```

Comment:
```suggestion
        return (2 * np.arctan(focal_lengths * slit_vector)).reshape((x, 2))
```

No need to do the unit conversion through numpy. If the user needs to convert units from radians to degrees, it can be done using .to(unit.deg) (from astropy)